src/foam/parse/QueryParserUserTest.js

```diff
@@ -0,0 +1,62 @@
+foam.CLASS({
+  package: 'foam.parse',
+  name: 'QueryParserUserTest',

```

Comment:
Why is this called UserTest instead of just Test? What does User refer to?

Reply:
For the QueryParser, we need to define a ClassInfo. I use User as a ClassInfo to be able to parse the query dynamically.

Reply:
ok